docs/developers_guide/ocean/tasks/geostrophic.md

```diff
@@ -0,0 +1,65 @@
+(dev-ocean-geostrophic)=
+
+# geostrophic
+
+The {py:class}`polaris.ocean.tasks.geostrophic.Geostrophic`
+test performs a series of 24-day runs that advect a bell-shaped tracer blob
+around the sphere.  The resolution of the sphere varies (by default, between
+60 and 240 km).  Advected results are compared with a known exact solution to
+determine the rate of convergence.
```

Comment:
I believe this description is left over from the `cosine_bell` case

Reply:
Oops, thanks!